packages/core/admin/ee/server/controllers/workflows/index.js

```diff
@@ -151,6 +179,10 @@ module.exports = {
       workflowService.count(),
     ]);
 
+    if (!this.checkWorkflowContentTypesPermissions(ctx.state.userAbility, workflows)) {
+      return ctx.forbidden();
+    }
```

Comment:
if the user doesn't have the permission to read any of the content types, it will not be able to see the list of workflows? I find that quite a bit too limiting.

Have we discussed about sanitizing the content types the user doesn't have visibility on?

Reply:
I am not really in favor of this approach because it could be confusing for users. With this you would get a forbidden error when listing all errors without much explanation!